scripts/app_start.sh

```diff
@@ -7,7 +7,8 @@ source ${BASH_SOURCE%/*}/../configs/$DEPLOYMENT_GROUP_NAME-config.conf
 
 # Re-read supervisor config, and add new processes
 echo "Reloading supervisor"
-supervisorctl update $APP_NAME $APP_NAME-scheduler
+supervisorctl reread $APP_NAME $APP_NAME-scheduler
+supervisorctl start $APP_NAME $APP_NAME-scheduler
```

Comment:
IDK what it is about this API but I almost never get it right 😅 This seems correct to me after reading the docs you linked though!